cmd/node/config/external.toml

```
@@ -37,6 +37,10 @@
     # RequestTimeoutSec defines the timeout in seconds for the http client
     RequestTimeoutSec = 60
 
+    # MarshallerType is used to define the marshalled type to be used for inner
```

```suggestion
    # MarshallerType is used to define the marshaller type to be used for inner
```
can be done in next PR